locations.go

```diff
@@ -19,6 +19,13 @@ type Location struct {
 	IsDefault        bool              `json:"is_default"`
 	IsVisible        bool              `json:"is_visible"`
 	ComputeResources []ComputeResource `json:"compute_resources"`
+	AvailablePlans   []ShortPlan       `json:"available_plans"`
+}
+
+// ShortLocation represents only ID and name of location.
+type ShortLocation struct {
```

Comment:
Please make a different pull request and move it to the place where the full version of Locations was placed.